practice/timezone/timezone/logic.py

```diff
@@ -0,0 +1,49 @@
+"""Runs the logic for the timezone Django app."""
+
+import arrow
+import re
+import tzwhere
+from tzwhere import tzwhere
+tz = tzwhere.tzwhere()
+
+
+def get_time_now(timezone):
+    """Takes in the timezone as an agurment and returns the ISO 8601 string
+    format version of the current time for the given timezone.
+
+    If '' is passed
+    in as the timezone, the time is returned for UTC time.
```

Comment:
I would make this functionality two separate functions. It's not clear from the function signature what this function should do when a non-timezone is passed in. Having a separate `get_local_time_now()` or something is very explicit.